packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -232,4 +233,85 @@ describe("TinyliciousClient", () => {
         const obj = await map1.get("newpair-id").get();
         assert.ok(obj, "container added dynamic objects incorrectly");
     });
+
+    /**
+     * Scenario: test if connect()/disconnect() can manage a container's connection state
+     *
+     * Expected behavior: the container should be able to connect and disconnect. The connectionState
+     * value should be updated accordingly. No errors should be thrown.
+     */
+    it("can disconnect() and connect() a container", async () => {
+        const container = (await tinyliciousClient.createContainer(schema)).container;
+        await container.attach();
+        await new Promise<void>((resolve) => {
+            container.once("connected", () => {
+                resolve();
+            });
+        });
+
+        container.disconnect?.();
+        assert.strictEqual(container.connectionState, ConnectionState.Disconnected, "container can't disconnect");
+
+        container.connect?.();
+        await new Promise<void>((resolve) => {
+            container.once("connected", () => {
+                resolve();
+            });
+        });
+        assert.strictEqual(
+            container.connectionState, ConnectionState.Connected,
+            "container can't connect after calling disconnect()",
```

Comment:
Personal choice - I like test asserts that say what _should_ happen, not what _didn't_ happen. Makes the tests easier to read IMO, it's like a comment in the code while still making sense in the test output if it fails.  Up to you though - it's not _unclear_ as-is, just wanted to share the thought.